src/Compilers/CSharp/Test/Emit3/ExtensionTypeTests.cs

```diff
@@ -2355,46 +2355,9 @@ class UnderlyingClass : I { }
         // PROTOTYPE should parse but remain error
         var comp = CreateCompilation(src, targetFramework: TargetFramework.Net70);
         comp.VerifyDiagnostics(
-            // (3,20): error CS9314: No part of a partial extension 'R' includes an underlying type specification.
+            // (3,21): error CS9329: Extension cannot have a primary constructor.
```

Comment:
>// (3,21): error CS9329: Extension cannot have a primary constructor. [](http://example.com/codeflow?start=12&length=69)

Please verify that constructor is not added as a member to the type.

Reply:
added check.